<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeepMind | Task App</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>481</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Krub</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasksListWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>341</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font:12pt;</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
background-color:#4267B2;
color:white;
font:11pt;
border-left: none;
border-right: 3px solid rgb(104, 104, 104);
border-bottom: 5px solid rgb(104, 104, 104);
}

QPushButton:hover{
background-color:rgb(51, 80, 139);
border-left: none;
border-right: 3.5px solid rgb(104, 104, 104);
border-bottom: 4.5px solid rgb(104, 104, 104);
}

QPushButton:pressed{
background-color: rgb(0, 86, 129);
border-bottom: none;
}</string>
   </property>
   <property name="text">
    <string>Save Change</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:#4267B2;
border: none;
color:white;
border-left: none;
border-right: 2.5px solid rgb(104, 104, 104);
border-bottom: 4px solid rgb(104, 104, 104);

}

QPushButton:hover{
background-color:rgb(51, 80, 139);
border-left: none;
border-right: 2px solid rgb(104, 104, 104);
border-bottom: 4px solid rgb(104, 104, 104);
}

QPushButton:pressed{
background-color: rgb(0, 86, 129);
border-bottom: none;
}</string>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 24pt;
background-color:#4267B2;
color:white;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Keep Mind &lt;/span&gt;Task App&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Created: Sarunwit Nontasean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contactButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:#4267B2;
border: none;
color:white;
border-left: none;
border-right: 2.5px solid rgb(104, 104, 104);
border-bottom: 4px solid rgb(104, 104, 104);

}

QPushButton:hover{
background-color:rgb(51, 80, 139);
border-left: none;
border-right: 2px solid rgb(104, 104, 104);
border-bottom: 4px solid rgb(104, 104, 104);
}

QPushButton:pressed{
background-color: rgb(0, 86, 129);
border-bottom: none;
}</string>
   </property>
   <property name="text">
    <string>Contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>480</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:#4267B2;
border: none;
color:white;
border-left: none;
border-right: 2.5px solid rgb(104, 104, 104);
border-bottom: 4px solid rgb(104, 104, 104);

}

QPushButton:hover{
background-color:rgb(51, 80, 139);
border-left: none;
border-right: 2px solid rgb(104, 104, 104);
border-bottom: 4px solid rgb(104, 104, 104);
}

QPushButton:pressed{
background-color: rgb(0, 86, 129);
border-bottom: none;
}</string>
   </property>
   <property name="text">
    <string>About project</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
